components/PrimarySourceSetsComponents/AllSets/components/SetsList/SetsList.module.css

```
@@ -4,7 +4,7 @@
   margin: -0.2rem auto 1rem;
 
   /* Overwrite flexbox-grid with a thinner gutter */
-  & :global .row {
+  :global(.row) {
```

Stylelint (17.13.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)